appveyor.yml

```
@@ -27,13 +27,14 @@ install:
         throw "There are newer queued builds for this pull request, failing early." }
 
   # set up environment
-  - "set PATH=%MINICONDA%;%MINICONDA%\\Scripts;%PATH%"
+  - CALL "%MINICONDA%\\Scripts\\activate.bat"
   - conda config --set always_yes yes --set show_channel_urls true --set changeps1 no
-  - conda install conda=4.6
+  - conda update conda
   - conda info -a
   - conda env create --file="${ENV_FILE}"
 
 test_script:
+  - CALL "%MINICONDA%\\Scripts\\activate.bat"
```

For one thing, you need not run activate twice here.  You can just pass the env name in the one command:

```suggestion
  - CALL "%MINICONDA%\\Scripts\\activate.bat" test
```

and remove "activate test" below

You also need to be extremely cautious about what "activate" is doing.  It might be conda's activate script, but it might not be.  I don't see any sane reason why it would not be here, but stranger things have happened.